web/ce/hooks/use-additional-editor-mention.tsx

```
@@ -0,0 +1,20 @@
+import { TMentionSection } from "@plane/editor";
+// plane types
+import { TSearchResponse } from "@plane/types";
+
+export type TAdditionalEditorMentionHandlerArgs = {
+  response: TSearchResponse;
+  sections: TMentionSection[];
+};
+
+export const useAdditionalEditorMention = () => {
+  const updateAdditionalSections = useCallback((args: TAdditionalEditorMentionHandlerArgs) => {
+    const {} = args;
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)